cf/test/test_read_write.py

```diff
@@ -462,26 +445,15 @@ def test_read_write_string(self):
             self.assertTrue(f[j].data.equals(f[i].data, verbose=1),
                             "{!r} {!r}".format(f[j], f[i]))
 
+        f0 = cf.read(self.string_filename)
         for string0 in (True, False):
             for fmt0 in ('NETCDF4',
-                         'NETCDF3_CLASSIC',
-                         'NETCDF4_CLASSIC',
-                         'NETCDF3_64BIT',
-                         'NETCDF3_64BIT_OFFSET',
-                         'NETCDF3_64BIT_DATA'):
-                # print ('fmt0=', fmt0)
-                f0 = cf.read(self.string_filename)
+                         'NETCDF3_CLASSIC'):
```

Comment:
Is the removal of the loops to test the other formats ('NETCDF4_CLASSIC' etc.) intentional here? Raising this in case you removed them from the sequence for development only, such that we should put them back as they were.

Ditto for the equivalent removal just below in the change set.

Reply:
This was intentional, as the other omitted formats collapse into the `NETCDF3_CLASSIC` case - there is no code bypassed here, fortunately.

Reply:
Ah, fair. All good in that case. Thanks.